test.php

```diff
@@ -6,9 +6,6 @@
 
 // Initialize template engine https://twig.symfony.com/
 $loader = new FilesystemLoader('templates');
-$twig = new Environment($loader, [
```

Comment:
I removed it,and how do I return the value,I mean what should I put instead of twig variable in the return statement?